package/libstorage.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jul 10 10:07:26 UTC 2015 - ancor@suse.com
+
+- remove partition from efi before destroying it on GPT disks
+  (bnc#929677)
```

Comment:
Lets hope the bug tracking scripts can handle the bug split and know that #870211 is also fixed.